src/main/java/eu/dissco/backend/properties/FdoProperties.java

```diff
@@ -0,0 +1,24 @@
+package eu.dissco.backend.properties;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+
+@Validated
+@Data
+@RequiredArgsConstructor
+@ConfigurationProperties("fdo")
```

Comment:
Should we specify here that this is about the mjr?

Reply:
I don't think that's necessary. Currently the only objects managed by the backend that receive a handle are MJRs, but  might change. We've come close to linking the handle api with the backend a few times now, and we might decide more objects need handles down the line. I'd leave it for now.